tests/model_hub_tests/performance_tests/conftest.py

```diff
@@ -7,6 +7,39 @@
 from models_hub_common.utils import get_params
 from py.xml import html
 from models_hub_common.utils import round_num
+import json
+import requests
+
+
+def pytest_sessionstart(session):
+    print('pytest_sessionstart')
+    session.results = []
+
+
+def pytest_sessionfinish(session, exitstatus):
```

Comment:
Maybe it would be more maintainable to produce results.json and then send it as separate step in CI?
It allow us to determinate test and infrastructure errors.